portfolio/src/main/webapp/script.js

```diff
@@ -37,26 +37,49 @@ function getComments() {
     });
 }
 
-/** Creates an <ul> element containing comment data.*/
+/**
+ * Returns HTML smiley emoji based on the sentiment score.
+ */
+ function returnEmoji(score) {
+  if (1 >= score && score > 0.6) { 
+    return `Vibe Check: ${String.fromCodePoint(0x1F600)}\n`; // Very happy
+  }
+  else if (0.6 >= score && score > 0.2) {
+    return `Vibe Check: ${String.fromCodePoint(0x1F642)}\n`; // Happy
+  } 
+  else if (0.2 >= score && score >= -0.2) {
+    return `Vibe Check: ${String.fromCodePoint(0x1F610)}\n`; // Neutral
+  } 
+  else if (-0.2 > score && score >= -0.6) {
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks): Line break after the closing brace if that brace terminates a statement or the body of a function or class statement, or a class method. Specifically, there is no line break after the brace if it is followed by else, catch, while, or a comma, semicolon, or right-parenthesis.

Did the formatter do this? It should be `} else if {` and `} else {`

Reply:
Ah got it. I've used the formatter to fix this styling issue. 👍